engine/src/main/scala/cromwell/engine/workflow/ValidateActor.scala

```diff
@@ -78,7 +65,7 @@ class ValidateActor(wdlSource: WdlSource, workflowInputs: Option[WdlJson], workf
     * 2.) Raw inputs and it's coercion
     * 3.) Workflow Options
     * 4.) RuntimeAttributes w.r.t. a backend
-    * @return
+    * @return The result of the validation as instance of `ValidationResult`
     */
   private def validateAll(): Future[ValidationResult] = {
```

Comment:
This might be the PR for making this method work in a more standard way, i.e. the `Future` doesn't need to always be successful:

```scala
private def validateAll: Future[ValidationResult] = for {
  ...
} yield ValidationSuccess(...)
```

and above:

```scala

def respond(response: ValidationResult): Unit = {
  requester ! response
  context.stop(self)
}

validateAll recover { case t => ValidationFailure(t) } foreach respond
```